lib/helper.js

```diff
@@ -5,6 +5,21 @@ const hoek = require('@hapi/hoek');
 const dayjs = require('dayjs');
 const TEMPLATE_NAME_REGEX_WITH_NAMESPACE = schema.config.regex.FULL_TEMPLATE_NAME_WITH_NAMESPACE;
 const formatDate = dateTime => dayjs(dateTime).format('YYYY-MM-DD');
+const PR_JOB_NAME_REGEX = schema.config.regex.PR_JOB_NAME;
+
+/**
+ * Get the original job name and PR number from full PR job name
+ * @param  {String} name Full PR job name (e.g.: PR-100:main)
+ * @return {Object}      PR number and job name
+ */
+function parsePRJobName(name) {
```

Comment:
Looks like we have similar implementation in [`pipeline` model](https://github.com/screwdriver-cd/models/blob/master/lib/pipeline.js#L430). 

May be we could consolidate and move these to `job` model. Similar to [`isPR()` and `prNum`](https://github.com/screwdriver-cd/models/blob/master/lib/job.js#L134) in the `job` model.

Reply:
Yeah that was the original intent of the helper function
was going to do that in a follow up PR

Reply:
This function need not be in generic helper. This seems to be specific to `job`, can we abstract this into `job` model and expose derived attributes or methods on the `job` model.

This would remove redundant implementation we have in other models
- [`pipeline`]([`pipeline`](https://github.com/screwdriver-cd/models/blob/master/lib/pipeline.js#L430))
- [`build`](https://github.com/screwdriver-cd/models/blob/master/lib/build.js#L33)

Reply:
This function need not be in generic helper. This seems to be specific to `job`, can we abstract this into `job` model and expose derived attributes or methods on the `job` model.

This would remove redundant implementation we have in other models
- [`pipeline`]([`pipeline`](https://github.com/screwdriver-cd/models/blob/master/lib/pipeline.js#L430))
- [`build`](https://github.com/screwdriver-cd/models/blob/master/lib/build.js#L33)